src/bls/bls_ies.cpp

```diff
@@ -44,8 +44,7 @@ bool CBLSIESEncryptedBlob::Decrypt(size_t idx, const CBLSSecretKey& secretKey, C
         return false;
     }
 
-    std::vector<unsigned char> symKey = pk.ToByteVector(false);
-    symKey.resize(32);
+    auto symKey = pk.ToBytes(false);
```

Comment:
nit: I think better to write directly type of `symKey`; it's not something super complicated but make code easier to read.

also consider {} initialization instead =

Reply:
Yes,  because it's easy to see, that there's no allocation, and it's raw bytes, no any wrappers or overhead. I guess it's still subjective and anyway it's an opinion